netlify-functions-ecommerce/.config/development.js

```diff
@@ -1,24 +1,11 @@
 'use strict';
-  
-const assert = require('assert');
-
-const astraUri = process.env.ASTRA_URI;
-
-if (!astraUri) {
-  assert.ok(process.env.STARGATE_BASE_URL, 'Must set STARGATE_BASE_URL environment variable');
-  assert.ok(process.env.STARGATE_AUTH_URL, 'Must set STARGATE_AUTH_URL environment variable');
-  assert.ok(process.env.STARGATE_USERNAME, 'Must set STARGATE_BASE_URL environment variable');
-  assert.ok(process.env.STARGATE_PASSWORD, 'Must set STARGATE_BASE_URL environment variable');
-}
 
 module.exports = Object.freeze({
-  stargateBaseUrl: process.env.STARGATE_BASE_URL,
-  stargateAuthUrl: process.env.STARGATE_AUTH_URL,
-  stargateUsername: process.env.STARGATE_USERNAME,
-  stargatePassword: process.env.STARGATE_PASSWORD,
-  astraUri: process.env.ASTRA_URI,
+  astraUri: 'http://127.0.0.1:8180/v1/cycling',
```

Comment:
in my opinion, we should not reference "Astra" except where we really mean astra. In this case, I think it should be something like `stargateJsonUri` which could be self managed or astra location.